proto/pom.xml

```diff
@@ -25,6 +25,11 @@
 				<groupId>org.apache.felix</groupId>
 				<artifactId>maven-bundle-plugin</artifactId>
 				<extensions>true</extensions>
+                <configuration>
```

Comment:
The alignment looks wrong here because this file has a mix of tabs and spaces. Let me know if I should format the entire file for consistency.